NFC_EEPROM/source/main.cpp

```diff
@@ -37,6 +37,16 @@ using mbed::nfc::ndef::common::span_from_cstr;
 /* URL that will be written into the tag */
 const char url_string[] = "mbed.com";
 
+#ifdef TARGET_MM
```

Comment:
No, this has nothing to do in the main file. Hide these details in specific target code.